presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -25,6 +25,9 @@ services:
   presto-master:
     extends:
       service: java-8-base
+    #image: 'teradatalabs/centos6-java8-oracle-mapr:${DOCKER_IMAGES_VERSION}'
+    # Will replace the image once we have that on docker-hub
```

Comment:
Replace what? I am not following this comment, are you talking about the commented image line above this?